spec/lib/programme_to_iati_status_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "#programme_status_to_iati_status" do
+  context "when the user sets the programme status for an activity" do
+    it "the correct iati status will be set automatically" do
```

Comment:
This read strangely when you include the `it`.

How about `it "sets the correct IATI status automatically"` ?

Reply:
I'll use your suggestion here. Sounds definitely better!